workers/ohsome_quality_analyst/indicators/ghs_pop_comparison_buildings/indicator.py

```diff
@@ -50,7 +51,20 @@ def yellow_threshold_function(self, pop_per_sqkm) -> float:
         return 0.75 * np.sqrt(pop_per_sqkm)
 
     async def preprocess(self) -> None:
-        pop_count, area = await self.get_zonal_stats_population()
+        raster = os.path.abspath(
+            os.path.join(
+                os.path.dirname(__file__),
+                "..",
+                "..",
+                "..",
+                "..",
+                "data",
+                "GHS_POP_E2015_GLOBE_R2019A_54009_1K_V1_0.tif",
+            )
+        )
+        stats = rasterstats.zonal_stats(self.feature, raster, stats="sum")
```

Comment:
What you want to do is to use the provided function `get_raster_dataset`. This will take care of path handling.
```suggestion
        from ohsome_quality_analyst.utils.definitions import get_raster_dataset
        raster = get_raster_dataset("GHS_POP_R2019A")
        stats = rasterstats.zonal_stats(self.feature, raster, stats="sum")
```